macros/unimplementedGeneric.ejs

```diff
@@ -63,7 +63,7 @@ switch($0) {
     %><span class="inlineIndicator unimplemented unimplementedInline"><%-str%></span><%
     break;
   case 'header':
-    %><div class="overheadIndicator unimplemented unimplementedHeader">
+    %><div class="blockIndicator unimplemented unimplementedHeader">
```

Comment:
Did not find any pages using this macro:
https://developer.mozilla.org/en-US/search?topic=none&kumascript_macros=secureContextGeneric&locale=*

Reply:
This is used inside the [`{{Unimplemented_Inline}}`](https://github.com/mdn/kumascript/blob/master/macros/unimplemented_inline.ejs) macro.